admin_app/src/components/ContentModal.tsx

```diff
@@ -137,9 +141,9 @@ const ContentViewModal = ({
                 <RemoveRedEye fontSize="small" color="disabled" />
                 <Typography variant="body2">_</Typography>
```

Comment:
We may want to populate this when a content is returned. Probably just for embeddings search

Reply:
Even for generated response we give reference content, so we can still collect info there perhaps?

Reply:
Should it just be that every time a content is fetched (be it for embedding search or as part of an llm-response call) this should tick up by one?

Reply:
Not sure. We can discuss and add it in a different PR